nvm.sh

```diff
@@ -2355,6 +2431,16 @@ nvm() {
   do
     case $i in
       '-h'|'help'|'--help')
+        NVM_NO_COLORS=""
+        for i in "$@"
+        do
+          if [[ $i == "--no-colors" ]]
```

Comment:
this is bash-specific syntax which won't work in all supported shells.
```suggestion
          if [ "${i}" = '--no-colors' ]
```

Reply:
Added `{}` and changed double-quotes to single-quotes